stacks_queues/stack.py

```diff
@@ -21,18 +21,19 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        return self.store.remove_first()
+
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.empty()
 
     def __str__(self):
         """ Returns the Stack in String form like:
             [3, 4, 7]
             Starting with the top of the Stack and
             ending with the bottom of the Stack.
         """
-        pass
+        return self.store.__str__()
```

Comment:
Dunder functions like `__str__` are what's known as operator overloading. `str` is a built-in function in Python that will turn a string version of any object passed to it, but we can redefine it in our classes to do exactly what we would like (thereby _overloading_ the original functionality of the `str` function/operator). When we implement our redefinition, we indicate we are doing operator overloading with the dunder `__str__`, however when we _use_ the overloaded operator, we do not need the dunder (though it will still work). 
```suggestion
        return str(self.store)
```